lib/vmpooler/api/v1.rb

```diff
@@ -5,48 +5,59 @@ class V1 < Sinatra::Base
     api_prefix  = "/api/v#{api_version}"
 
     helpers do
-      def hostname_shorten(hostname)
-        if $config[:config]['domain'] && hostname =~ /^\w+\.#{$config[:config]['domain']}$/
-          hostname = hostname[/[^\.]+/]
-        end
+      def get_capacity_metrics()
+        capacity = {
+          current: 0,
+          total: 0,
+          percent: 0
+        }
 
-        hostname
-      end
+        $config[:pools].each do |pool|
+          pool['capacity'] = $redis.scard('vmpooler__ready__' + pool['name']).to_i
 
-      def validate_date_str(date_str)
-        /^\d{4}-\d{2}-\d{2}$/ === date_str
-      end
+          capacity[:current] += pool['capacity']
+          capacity[:total] += pool['size'].to_i
+        end
 
-      def mean(list)
-        s = list.map(&:to_f).reduce(:+).to_f
-        (s > 0 && list.length > 0) ? s / list.length.to_f : 0
-      end
-    end
+        if capacity[:total] > 0
+          capacity[:percent] = ((capacity[:current].to_f / capacity[:total].to_f) * 100.0).round(1)
+        end
 
-    get "#{api_prefix}/status/?" do
-      content_type :json
+        capacity
+      end
 
-      result = {
-        status: {
-          ok: true,
-          message: 'Battle station fully armed and operational.'
-        },
-        capacity: {
-          current: 0,
-          total: 0,
-          percent: 0
-        },
-        clone: {
+      def get_clone_metrics(date)
+        clone = {
           duration: {
             average: 0,
             min: 0,
-            max: 0
+            max: 0,
+            total: 0
           },
           count: {
             total: 0
           }
-        },
-        queue: {
+        }
+
+        clone[:count][:total] = $redis.hlen('vmpooler__clone__' + date).to_i
```

Comment:
Anything that uses one of the helper functions should be passing the `date` param as a string.